cmd/bosun/main.go

```diff
@@ -155,16 +154,9 @@ func main() {
 	}()
 	if *flagWatch {
 		watch(".", "*.go", quit)
-		watch(filepath.Join("web", "static", "templates"), "*.html", quit)
+		watch(filepath.Join("web", "static", "templates"), "*.html", web.RunEsc)
```

Comment:
Why did this change? It quits when index.html changes, which needs to be done because it gets loaded only once on start.